lib/generators/lightning_cad/templates/.eslintrc.js

```diff
@@ -0,0 +1,123 @@
+export default {
```

Comment:
Probably better to just use the ESLint config provided by the generic (non-L.CAD) generator. There might be changes from projects that could be good to bring in to our base config.